Woof/WoofTests/Helper/DataTransformerTests/NormalizeToZeroToFiveRangeTests.swift

```diff
@@ -2,34 +2,34 @@ import XCTest
 
 final class NormalizeToZeroToFiveRangeTests: XCTestCase {
     func testConvertValueReturnsZeroWhenNegativeValueProvided() {
-        XCTAssertEqual(DataTransformer.normalizeToZeroToFiveRange(-25.0), 0)
+        XCTAssertEqual(DataTransformer.normalizeToZeroToFiveRange(-25.0), .unavailable)
```

Comment:
Update the method name, please.
Review other test method names as well.

Reply:
Fixed.